docs/getting-started/server/guide.md

```diff
@@ -294,6 +294,29 @@ attempting to "Build and Run the Server".
 
 </Bitwarden>
 
+## Configure NuGet with GitHub Packages
+
+Server-side projects and solutions may use
+[Bitwarden-shared .NET extension libraries hosted on GitHub Packages](https://github.com/orgs/bitwarden/packages?repo_name=dotnet-extensions)
+and these packages require authentication for access.
+
+First, [generate](https://github.com/settings/tokens/new) a GitHub personal access token (classic)
+with the `packages:read` scope only. You can set an expiration date but it may be easier to leave it
+without one considering the scope. Copy the token value and run:
```

Comment:
Probably a good idea to link to here for further reading: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry